pipeline/load/display_metrics.py

```diff
@@ -1,59 +1,34 @@
-import operator
+from typing import Dict, List
 
 import prefect
 from prefect import task
 
-from pipeline.types import MatchingMetrics, MatchingOutput, User
+from pipeline.types import MatchingMetrics, MatchingOutput, User, UserId
+
+
+def render_counts_per_user(
+    users: List[User], counts: Dict[UserId, int]
+) -> List[str]:
+    """Helper method to get lines of counts per user in descending order."""
+    display_names = {user.uid: user.displayName for user in users}
```

Comment:
Instead of doing a complicated key replace, we can use a dictionary comprehension to get the name for each user ID, then get it simply by calling:

```python
name = display_names.get(uid)
```

We do not have to worry about cases where the user ID does not have a name, since that would be caused by an error elsewhere in the codebase.